contracts/test/PDPService.t.sol

```diff
@@ -348,21 +376,65 @@ contract PDPServiceProofSetMutateTest is Test {
         recordAssert.expectRecord(PDPListener.OperationType.ADD, setId);
         uint256[] memory toRemove = new uint256[](1);
         toRemove[0] = 1;
-        pdpService.removeRoots(setId, toRemove);
-
         toRemove[0] = 0;
```

Comment:
One of these two lines is now redundant, and possibly making the test pointless. It's not obvious from the test itself which one (the reader needs to know what the first root ID allocated is).

Should this just do the original pattern of two (schedule) removals?

Reply:
And yeah I think this was just a mistake